testing/kuttl/e2e/exporter-password-change/files/update-monitoring-password-checks.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: exporter-password-change-monitoring
+  labels:
+    postgres-operator.crunchydata.com/cluster: exporter-password-change
+    postgres-operator.crunchydata.com/role: monitoring
+  ownerReferences:
+  - apiVersion: postgres-operator.crunchydata.com/v1beta1
+    blockOwnerDeletion: true
+    controller: true
+    kind: PostgresCluster
+    name: exporter-password-change
+data:
+  # ensure the password is encoded to 'password'
+  password: cGFzc3dvcmQ=
+---
+# TODO: Check that password is set as a file
```

Comment:
Is this TODO still valid?

Reply:
I don't think we need that. We could check that the volume is mounted as expected but that would require checking a list :thinking: 

In the assert step we confirm that the contents of the file are what we expect - i think this accomplished the same thing